### PR TITLE
Improvements for launch

### PR DESCRIPTION
**Pull request by aduggin**

### CODE REVIEW

File: app/views/components/_phase_banner.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="phase-banner">
+  <p>
+    <strong class="phase-tag">BETA</strong>
+          <span>This is a new service under development – your
+            <a href="mailto:beta-feedback@pensionwise.gov.uk?Subject=Feedback%20for%20Pension%20Wise%20Beta%20Service">
```
Comment (andrewgarner) [inline app/views/components/_phase_banner.html.erb]: Wouldn't a subject that calls out the specific part of the service be more appropriate? Maybe "Feedback for registering interest in Pension Wise guidance appointments". There may be many parts of the service that are in BETA.

File: app/views/components/_phase_banner.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="phase-banner">
+  <p>
+    <strong class="phase-tag">BETA</strong>
+          <span>This is a new service under development – your
```
Comment (stevenwilkin) [inline app/views/components/_phase_banner.html.erb]: Indented too far?

File: app/assets/stylesheets/layout/_grid.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+.l-column-third {
+  @include grid-column(1/3);
+}
+
+.l-column-two-thirds {
+  @include grid-column(2/3);
+}
+
+
```
Comment (stevenwilkin) [inline app/assets/stylesheets/layout/_grid.scss]: Extra empty line

Comment (jiannis) [inline app/assets/stylesheets/layout/_grid.scss]: Oh dear!

File: app/views/components/_phase_banner.html.erb (hunk 2 above)
Comment (jiannis) [inline app/views/components/_phase_banner.html.erb]: Welcome @stevenwilkin !

**Pull request merged**